watermark.php

```diff
@@ -169,6 +196,12 @@ private function _postValidation()
             $this->_postErrors[] = $this->trans('Opacity is not in allowed range.', [], 'Modules.Watermark.Admin');
         }
 
+        if (empty($resize)) {
+            $this->_postErrors[] = $this->trans('Resize required.', [], 'Modules.Watermark.Admin');
+        } elseif ($resize < 1 || $resize > 100) {
+            $this->_postErrors[] = $this->trans('Resize not allowed range.', [], 'Modules.Watermark.Admin');
```

Comment:
```suggestion
            $this->_postErrors[] = $this->trans('Resize value not in the range (1 - 100).', [], 'Modules.Watermark.Admin');
```
Something like this maybe?